PCIe 6.0 Packet Generator and Exerciser/PCIe 6.0 Packet Generator and Exerciser/layers/receiver.cpp

```diff
@@ -0,0 +1,11 @@
+
+#include "receiver.h"
+//nopflit
+void Receiver::receiveInit(Flit flit) {
+	this->layers.receiveNOPFlit(flit,this->globals.P_SHARED_CREDIT_LIMIT,this->globals.NP_SHARED_CREDIT_LIMIT,this->globals.CPL_SHARED_CREDIT_LIMIT,this->globals.Fl1,this->globals.Fl2);
+}
+
+//payloadflit
+void Receiver::receiveFlit(Flit flit) {
+	this->layers.receivePayloadFlit(flit, this->sendOn, this->globals.P_CREDIT, this->globals.NP_CREDIT, this->globals.CPL_CREDIT);
+}
```

Comment:
These functions aren't building because the layers wrapper functions are not implemented yet. Also, the signatures will change as we added dedicated credits. Please mark the wrappers functions as blocking you on Jira so you are notified whenever they are ready